### PR TITLE
1st Capstone project on Global Impex Expo

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Designing a mobile version of the application.
Milestone 2: Designing a desktop version of the application.
Milestone 3: Add Javascript objects to fetch the data.
Milestone 4: Add More Button functionality.

I have used Bootstrap to develop this application and made some changes in the design as the topic of the app is different.  

Loom Video: https://www.loom.com/share/2555727ca09e4077b79eb92f9debfa7a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ZawadiSumba66) [inline README.md:68]: - [ ] Kindly fix the MIT License link in your README file to maintain the professionalism of your documentation. It currently returns a page not found.

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Hector096/Conference-App/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (ZawadiSumba66) [inline README.md:59]: - [ ] In the acknowledgments section you are required to give credits to the author of the design as stated in the requirements: 👇 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="container-fluid main-bg">
+            <div class="container d-flex flex-column main-heading">
+                <h1 class="mt-3">"Hello! Sharing World"</h1>
+                <h1 class="display-1">Aanchal Global<br> Impex Expo <br>2021</h1>
```
Comment (ZawadiSumba66) [inline index.html]: - [ ] This text ought to have a bolder font weight as compared to the design to the design given:
yours:
image
========================================================================================
design given:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+                    </ul>
+                    <div class="d-flex justify-content-center">
+                        <button type="button" class="btn btn-outline-primary nav-btn1"> Get In Touch</button>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="container-fluid main-bg">
```
Comment (ZawadiSumba66) [inline index.html:69]: - [ ] Kindly add more padding at the bottom of this section to match the given design.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Dolores, autem fuga explicabo nobis
+                        dolorum ipsam vitae veniam blanditiis, eum quam et iusto porr</p>
+                </div>
+                <h2 class="pt-3 text-dark font-weight-bold"><strong>2021.10.15 ~ 16(FRI)</strong> </h2>
+                <h4 class="pt-2 text-secondary">@National Export Council of India, Export HUB and more </h4>
+            </div>
+
+        </section>
+        <section class="container-fluid products-box d-flex flex-column justify-content-center align-items-center">
```
Comment (ZawadiSumba66) [inline index.html]: - [ ] For the `Our Products` section, kindly follow the design as seen from this screenshot: 👇 
image

- [ ] In this section it should have five individual articles, each with an image, title, and description, and a line of text at the bottom as seen from the image above. 
- [ ] In the mobile view, remember to design this section in such a way that individual articles have their elements organized in a horizontal row, with the image at the left, title in the center, and description at the right.
As seen in the screenshot below from the design:
image

File: app.js [hunk 6]
```diff
@@ -0,0 +1,149 @@
+const featuredContainer = document.getElementById('featured-container');
+const productContainer = document.getElementById('products-list');
+const featuredContainer2 = document.getElementById('featured-container2');
+const productContainer2 = document.getElementById('products-list2');
+const productBtn = document.getElementById('load-product-button');
+const postBtn = document.getElementById('load-postt-button');
+
+const myProducts = [
+  {
+    name: 'Spices', description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.', imgUrl: 'https://images.unsplash.com/photo-1588088644163-8e743bcc01ef?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=752&q=80',
```
Comment (ZawadiSumba66) [inline app.js]: - [ ] It'd be awesome if you could reduce the size of each image in this section to match the design given. 💪

File: about.html [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="container-fluid main-bg">
+            <div class="container d-flex flex-column align-items-center main-heading">
+                <h1 class="pt-5">"Hello! Sharing World"</h1>
+                <h1 class="display-1 text-center">Aanchal Global<br> Impex Expo 2021</h1>
```
Comment (ZawadiSumba66) [inline about.html]: - [ ] As requested about the font-weight in the index HTML file kindly change it also here to match the design.

File: about.html [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+                        repellendus ab cumque eum iusto, ea tempore veritatis enim voluptates, aliquam eius animi?
+                        Officia expedita laborum reiciendis quia, nemo porro temporibus earum </p>
+                </div>
+                <p class="text-center pt-3">Please contact us per Email for any further queries</p>
+                <strong class="text-center pb-3">hello@aanchalenterprises.com</strong>
+            </div>
+
+        </section>
+        <section class="container-fluid bg-white d-flex flex-column justify-content-center">
+            <h2 class="mt-5 about-heading text-center">Aanchal Enterprises Global Export Summit 2021</h2>
```
Comment (ZawadiSumba66) [inline about.html:102]: - [ ] For all the headings on this page I would like you to consider incrementing the font-weight as can observe from the design given

File: index.html [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+            <button type="button" class="btn btn-outline-primary mb-5 d-lg-none d-md-block nav-btn" id="load-postt-button">
+                Load More Posts
+            </button>
+
+        </section>
+
+        <section class="container-fluid products-box d-flex flex-column justify-content-center">
+            <h2 class="mt-5 product-heading">Partner</h2>
+            <div class="container p-5">
+                <div class="partner">
```
Comment (ZawadiSumba66) [inline index.html]: - [ ] The partner logos here look awesome on desktop. However, I would like you to consider changing it to the design as this screenshot from the design on mobile view:
image

File: index.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+        integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="style.css">
+
+    <script src="https://kit.fontawesome.com/a03b3724cb.js" crossorigin="anonymous"></script>
+
+    <title>2021 CC global Summit</title>
+</head>
+
+<body>
```
Comment (ZawadiSumba66) [inline index.html:20]: - [ ] Let's make sure we have only one `h1` tag on each page both the about and index for  SEO reasons and to make your code look more semantic.

File: style.css [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+  transition: 0.5s;
+  background-color: var(--hoverColor);
+  padding: 8px;
+}
+
+#nav-cc-btn {
+  margin: auto;
+}
+
+@media (min-width: 768px) {
```
Comment (ZawadiSumba66) [inline style.css:224]: - [ ] You are encouraged to use mobile-first approach for your media queries i.e 
media queries are used to change the design from mobile to desktop, not the other way around.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi  @Hector096 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Davidosky007) [reply index.html]: * Fixed 👍

File: index.html (hunk 10 above)
Comment (Davidosky007) [reply index.html:20]: * Fixed 👍

File: about.html (hunk 8 above)
Comment (Davidosky007) [reply about.html:102]: * Fixed 👍

File: index.html (hunk 5 above)
Comment (Davidosky007) [reply index.html]: * Fixed 👍

File: index.html (hunk 4 above)
Comment (Davidosky007) [reply index.html:69]: * Fixed 👍

File: app.js (hunk 6 above)
Comment (Davidosky007) [reply app.js]: * Fixed 👍

**Comment by Davidosky007** [comment]
## Hi, @Hector096 @ZawadiSumba66 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**